server/src/routes/strategies.ts

```diff
@@ -102,18 +111,20 @@ export class GroundTruthStrategy extends OAuthStrategy {
                     variables
                 })
             };
-
+            console.log("ok")
             await request(options, async (err, res, body) => {
                 if (err) { return console.log(err); }
                 if (JSON.parse(body).data.search_user.users.length > 0) {
                     confirmed = JSON.parse(body).data.search_user.users[0].confirmed;
                 }
-                if (confirmed) {
+                console.log("ok" + confirmed)
+                if (true) {
```

Comment:
I know this is a temporary workaround, but remove (or comment) the conditional and else block here since it's impossible to reach the else branch.